### ISSUE TITLE
Incremental update engine.

### ISSUE THREAD

**Issue by sdlmw** [pytorch/fairseq#2349]
Hi ,

Does Fairseq support incremental updates? If so, is there relevant documentation.

thanks

**Comment by myleott** [comment]
What are incremental updates?

We support delayed updates (i.e., accumulate multiple batches of gradients before each update), is that what you mean? This is documented here: https://fairseq.readthedocs.io/en/latest/getting_started.html#large-mini-batch-training-with-delayed-updates
